udata/commands/__init__.py

```diff
@@ -1,109 +1,141 @@
 # -*- coding: utf-8 -*-
-from __future__ import print_function, unicode_literals
+from __future__ import unicode_literals, print_function
 
-import logging
 import os
+import click
 import sys
+import logging
+import pkg_resources
 
-from os.path import join, dirname, splitext, basename
 from glob import iglob
 
-from flask_script import Manager
-from flask_script.commands import Clean, ShowUrls, Server, InvalidCommand
-
+from flask.cli import FlaskGroup, shell_command, ScriptInfo
 from udata.app import create_app, standalone, VERBOSE_LOGGERS
 
-# TODO: Switch to flask.cli and drop flask-script dependency
+log = logging.getLogger(__name__)
 
-# Expect an utf8 compatible terminal
-reload(sys)
-sys.setdefaultencoding('utf8')
```

Comment:
Nop, this is handled by our code (explicit `encode`/`decode`) + testing.
But I removed these lines because they introduce more errors than they fix.
Plus, it is not recommended to do that as it breaks some Python internals.